### ISSUE TITLE
pkg-config support

### ISSUE THREAD

**Issue by hacker1024** [leighleighleigh/JCAN#20]
It'd be great to add support for [pkg-config](https://www.freedesktop.org/wiki/Software/pkg-config) to make it easier to use the library with build systems like CMake.

**Comment by leighleighleigh** [comment]
Good idea - I will look into it when I spring-clean this repo sometime this week/weekend.
